parts/linux/cloud-init/artifacts/mariner/cse_install_mariner.sh

```diff
@@ -27,11 +27,9 @@ installDeps() {
 
 installKataDeps() {
     if [[ $OS_VERSION != "1.0" ]]; then
-      for dnf_package in kernel-mshv hvloader mshv-bootloader-lx mshv cloud-hypervisor cloud-hypervisor-cvm moby-containerd-cc kata-containers kata-containers-cc kernel-uvm kernel-uvm-cvm; do
-        if ! dnf_install 30 1 600 $dnf_package; then
-          exit $ERR_APT_INSTALL_TIMEOUT
-        fi
-      done
+      if ! dnf_install 30 1 600 kata-packages-host; then
```

Comment:
assuming the answer is no, but will we need to be sure to wait for your PR to update the UVM signing build to merge and rollout before merging this PR in? 

we've ran into issues in the past where new artifacts published by the signing build weren't picked up in time for official releases which ended up causing problems

Reply:
Good point. No, we don't need to wait for updates on the UVM build and sign pipelines. They will produce the same artifacts before and after the update.

All the dependencies (kata-packages-host here) are on Azure Linux PMC so we are good to go.